mitodl/mitxpro
# Initial design and flow for DC download certificates

#### Pre-Flight checklist


#### What are the relevant tickets?
#2152 

#### What's this PR do?
Updates the UX to support the digital credentials on the dashboard for certificates.

#### How should this be manually tested?
- Create Program and Course certificates for your user after enrollment.
- Set/Enable `FEATURE_DIGITAL_CREDENTIALS ` flag.
- You should be able to see `Digital Credentials` link on the dashboard
- Click on the link and you should be able to verify the designs and functionality from the invasion links provided in the associated.


#### Screenshots (if appropriate)
**Dashboard(Desktop)**
<img width="1440" alt="Screenshot 2021-03-25 at 9 28 08 PM" src="https://user-images.githubusercontent.com/34372316/112508258-3736d400-8db1-11eb-8967-ea41169fa7eb.png">

**Modal (Desktop)**
<img width="1437" alt="Screenshot 2021-03-25 at 9 28 00 PM" src="https://user-images.githubusercontent.com/34372316/112508230-31d98980-8db1-11eb-8209-17073d2778ed.png">

**Dashboard(Mobile)**
<img width="499" alt="Screenshot 2021-03-25 at 9 28 23 PM" src="https://user-images.githubusercontent.com/34372316/112508274-39009780-8db1-11eb-8c4e-34f9151eda3a.png">

**Modal (Mobile)**

<img width="494" alt="Screenshot 2021-03-25 at 9 28 34 PM" src="https://user-images.githubusercontent.com/34372316/112508295-3bfb8800-8db1-11eb-99cb-d1c3d8715dac.png">
